src/orquestra/quantum/api/wavefunction_simulator_contracts.py

```diff
@@ -0,0 +1,228 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
+################################################################################
+from functools import partial
+from itertools import chain
+
+# Test values of expectation values
```

Comment:
[question] Do you think these comments are necessary?

Reply:
As above, removed in [8d9349a](https://github.com/zapatacomputing/orquestra-quantum/pull/66/commits/8d9349ae5d02f624484837a822a7ff80aa2914ef)